snapcraft/extensions/kde_neon_6.py

```diff
@@ -280,6 +356,10 @@ def get_parts_snippet(self) -> Dict[str, Any]:
         # https://github.com/canonical/snapcraft-desktop-integration
         source = get_extensions_data_dir() / "desktop" / "kde-neon-6"
```

Comment:
I guess the source should be changed to `command-chain` instead of `kde-neon-6`

Reply:
This never actually gets used.

Reply:
That wouldn't be very useful for kde as that runs

set -- "${SNAP}/gnome-platform/command-chain/desktop-launch" "$@"
# shellcheck source=/dev/null
source "${SNAP}/snap/command-chain/run"

Reply:
I made command-chain-kde to resolve the conflict. thanks!